linux/open_vm_tools/install_ovt.yml

```diff
@@ -13,7 +13,11 @@
   block:
     - name: "Add a local package repository from ISO image for {{ guest_os_ansible_distribution }}"
       include_tasks: ../utils/add_local_dvd_repo.yml
-      when: guest_os_ansible_distribution in ['SLES', 'SLED', 'RedHat']
+      when: >
+        guest_os_ansible_distribution in ['SLES', 'SLED', 'RedHat'] or
+        (guest_os_ansible_distribution in ['Rocky', 'AlmaLinux'] and 
```

Comment:
Here doesn't need to check version, just guest_os_ansible_distribution is enough.